core/sql/optimizer/BindRelExpr.cpp

```diff
@@ -13445,6 +13435,14 @@ RelExpr * GenericUpdate::bindNode(BindWA *bindWA)
   if (getOperatorType() == REL_UNARY_UPDATE ||
       getOperatorType() == REL_UNARY_DELETE) {
 
+    if (getTableDesc()->getNATable()->isHiveTable())
+      {
+        *CmpCommon::diags() << DgSqlCode(-4223)
+                            << DgString0("Update/Delete on Hive table is");
+        bindWA->setErrStatus();
+        return this;
+      }
+
```

Comment:
I'm curious why this change was made. Do we support other kinds of delete and update operations on Hive tables?